README.md

```diff
@@ -115,6 +115,20 @@ app.listen(80, function () {
 })
 ```
 
+If you want to enable access using CuRL or any other REST Tool / Server without origin, use `corsOptions` like this:
```

Comment:
I've never seen curl written that way before.  Can the capitalization just match what is used on the tool's site? https://curl.haxx.se

Reply:
Done.